### PR TITLE
Database Perfomance Audit

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of the pull request

- [ ] Add an 
analyse-1
analyse-2
analyse-3
analyse-4
analyse-5
analyse-6
email column to the owners' table
- [ ] carry out three different queries
- [ ] Carryout Explain Analyse query plan for those queries and take a screenshot
- [ ] Carryout indexing for queries to reduce query time. take a screenshot for them.
- [ ] Modify ReadME file.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -74,8 +74,35 @@ SELECT COUNT(*) FROM visits JOIN animals ON visits.animal_id = animals.id JOIN v
...
 
 
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM  visits WHERE  animal_id = 4;
 
+SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT  * FROM  visits WHERE vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+EXPLAIN ANALYZE SELECT  * FROM owners where  email = 'owner_18327@mail.com';
```
Comment (gemmen29) [inline queries.sql:89]: - [ ] Kindly make the execution time for all queries before the optimization exceeds or equal `1000ms` as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Kaweesi-Matia) [reply queries.sql:89]: Below are the s
a-1
screenshots
a-2
a-3
a-4
a-5
a-6
 for the required changes

**Review by alexander16108 — APPROVED** [review]
Hi @Kaweesi-Matia  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**